sdk/storage/azure-storage-blob/tests/test_quick_query.py

```diff
@@ -228,7 +228,8 @@ def test_quick_query_iter_records_with_headers(self, resource_group, location, s
         blob_client.upload_blob(CSV_DATA, overwrite=True)
 
         input_format = DelimitedTextDialect(has_header=True)
-        reader = blob_client.query_blob("SELECT * from BlobStorage", blob_format=input_format)
+        output_format = DelimitedTextDialect(has_header=False)
```

Comment:
do we have coverage for `output_format = DelimitedTextDialect(has_header=True)` ?

Reply:
good question, the test failed because we default the output_format to input_format which means has_header=True such that the assertion failed. So I explicitly set has_header=True to make the original assertion pass.
I think we should keep the original setting as well, let me add a test.

Reply:
yes,we should add coverage in this case.

Reply:
Just a note: I added the tests for has_header=True, just accidentally amended to the existing commit.